McCorrections/src/LeptonWeights.cpp

```diff
@@ -159,25 +174,18 @@ double LeptonWeights::GetTriggerEfficiency(EventInfo& eventInfo, bool isData) co
             double tau_eff = 1;
             tau_eff = isData ?
                 tauTriggerWeight_eTau->getEfficiencyData(static_cast<float>(eventInfo.GetLeg(2).GetMomentum().pt()),
-                                                                            eventInfo.GetLeg(2)->decayMode()) :
+                                                                            eventInfo.GetLeg(2)->decayMode(),
+                                                                            static_cast<int>(unc_scale)) :
```

Comment:
here and below: don't pass unc_scale to trigger SFs. In this way it is wrong. It will be implemented in the next PR.